ui/app/templates/vault/cluster.hbs

```diff
@@ -3,6 +3,40 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
+{{#if this.auth.showRootTokenWarning}}
+  <Modal
+    @title="Warning"
+    @onClose={{fn (mut this.auth.showRootTokenWarning) false}}
+    @isActive={{this.auth.showRootTokenWarning}}
+    @type="warning"
+  >
+    <section class="modal-card-body">
+      {{#if this.namespaceService.inRootNamespace}}
+        You have logged in with a root token. As a security precaution, this root token will not be stored by your browser
+        and you will need to re-authenticate after the window is closed or refreshed. Click
+        <b>confirm</b>
+        to continue.
+      {{else}}
+        As a security precaution, you will be logged out and need to log in again to navigate to the
+        <code>{{this.namespaceService.currentNamespacePath}}</code>
```

Comment:
So this modal shows before we've transitioned? Reading this it seems that `currentNamespacePath` was the namespace we're coming from, not going to - but I not sure how/when the service is set

Reply:
Yeah, I definitely agree that the wording is slightly confusing. The namespace is supposed to be the namespace we're going to. However, when the user logs into a specific namespace it would be what they're coming from?

Reply:
@hellobontempo I think maybe even keeping our old text makes the most sense here since this happens in two contexts, when a user logs in as root on the login page (from a root namespace or any namespace) and when the user selects a namespace in the namespace picker. 

The old message is: "You have logged in with a root token. As a security precaution, this root token will not be stored by your browser and you will need to re-authenticate after the window is closed or refreshed. Click confirm to continue."

Do you have any thoughts on this?